engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -13,16 +13,24 @@ import skuber.LabelSelector.IsEqualRequirement
 import skuber.apps.v1.Deployment
 import skuber.{Container, EnvVar, HTTPGetAction, LabelSelector, Pod, Probe, Volume}
 
+case class MountableResources(
+                              commonConfigConfigMap: String,
```

Comment:
is commonConfigConfigMap a whole config map ?

Reply:
it contains deployment.confg and scenario.json